snuba/web/query.py

```diff
@@ -236,6 +238,12 @@ def _format_storage_query_and_run(
     with sentry_sdk.start_span(description="create_query", op="db") as span:
         formatted_query = format_query(clickhouse_query, request_settings)
         span.set_data("query", formatted_query.structured())
+        span.set_data(
+            "query_size_bytes", _string_size_in_bytes(formatted_query.get_sql())
+        )
+        span.set_tag(
+            "query_size_group", get_query_size_group(formatted_query.get_sql())
+        )
```

Comment:
Apologies for my ignorance, are tags on spans actually searchable? I was under the impression they are not and only the ones on the actual transaction are.

Reply:
I'm not sure how to go about tagging the overall transaction and not just the span in this situation, I'll look into it.

Reply:
I think this is an alias for setting a tag on the transaction. I’d have to look at the code to be sure though.

Reply:
So apparently this is _sometimes_ setting a tag on the event and sometimes not. I would change this to `sentry_sdk.set_tag` to be sure.